robottelo/datafactory.py

```diff
@@ -32,32 +33,37 @@ def func_wrapper(*args, **kwargs):
 
 
 @datacheck
-def generate_strings_list(length=None, remove_str=None, bug_id=None):
+def generate_strings_list(length=None, exclude_types=None, bug_id=None,
```

Comment:
With the previous signature we had the ability of specifying length as an integer or a tuple (min, max) which is the format fauxfactory functions expect.

Adding min and max length arguments are redundant.